src/supermarket/app/helpers/markdown_helper.rb

```diff
@@ -1,73 +1,60 @@
+require 'html/pipeline'
+
 module MarkdownHelper
   #
-  # Make auto-links target=_blank
+  # Renders markdown as escaped HTML.
+  #
+  # @param text [String] markdown to be rendered
+  #
+  # @return [String] escaped HTML.
   #
-  class SupermarketRenderer < Redcarpet::Render::Safe
-    include ActionView::Helpers::TagHelper
+  def render_markdown(text)
+    context = {
+      base_url: Supermarket::Host.full_url,
+      http_url: Supermarket::Host.full_url
+    }
 
-    def initialize(extensions = {})
-      super extensions.merge(
-        link_attributes: { target: '_blank' },
-        with_toc_data: true
-      )
-    end
+    pipeline = HTML::Pipeline.new [
```

Comment:
Is this expesnive to build/should we cache the pipeline? (Not sure of the lifecycle of these things in supermarket)

Reply:
We were `new`ing the previous rendering class library. 🤔 I reckon it could be memoized or singleton'd, but I doubt we're spending much time in instantiating these during a web request.